sdk/storage/storage-queue/test/queueclient.spec.ts

```diff
@@ -110,7 +110,7 @@ describe("QueueClient", () => {
     const metadata = { key: "value" };
     const res2 = await queueClient.createIfNotExists({ metadata });
     assert.ok(!res2.succeeded);
-    assert.equal(res2.errorCode, "QueueAlreadyExists");
+    assert.equal((res2 as any).errorCode, "QueueAlreadyExists");
```

Comment:
Are we getting the wrong type of res2? Why did we have to cast?

Reply:
Reverted this.